src/glyphs.js

```diff
@@ -5,6 +5,8 @@ const ObjectId = require('mongodb').ObjectId;
 
 const db = require('./db');
 
+const sha256 = require('js-sha256');
```

Comment:
Did this getting added to `package.json` too?  We need to make sure that dependencies are all installed when a new user builds this.

Reply:
This slipped my mind, thanks for catching and taking care of it.